projects/fsastorefrontlib/src/occ/services/default-occ-product-config.ts

```diff
@@ -5,7 +5,7 @@ export const occProductConfig: OccConfig = {
     occ: {
       endpoints: {
         productSearch:
-          'products/search?fields=products(code,name,summary,defaultCategory(DEFAULT),price(FULL),images(DEFAULT)),pagination(DEFAULT),sorts(DEFAULT)',
+          'products/search?fields=products(code,name,summary,defaultCategory(DEFAULT),price(FULL),images(DEFAULT)),pagination(DEFAULT),sorts(DEFAULT),facets,breadcrumbs,freeTextSearch,currentQuery',
```

Comment:
Here I'm not sure if we need `freeTextSearch `, I think it works without that. `currentQuery ` should be enough to fix error from `filterForPage` method.